examples/sampleswagger.yaml

```diff
@@ -14,20 +14,20 @@ info:
     email: apiteam@wordnik.com
   license:
     name: Apache 2.0
-    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
```

Comment:
@aman-v-singh Why are we making this change?

Reply:
I locally ran `npm test` and the test cases were failing, showing there is a difference between actual and expected output.
This could have possibly happened as we changed the `js-yaml` version.

Reply:
Can we define what actually changed? This much changes means there might be some functionality that changes and we should be very aware of such change. It could suddenly break the existing working definitions for users.

Let's pinpoint the exact issue and if there's way to avoid it.

Reply:
This was possibly introduced here [here](https://github.com/nodeca/js-yaml/blob/master/CHANGELOG.md#:~:text=Added%20quotingType%20and%20forceQuotes%20options%20for%20dumper%20to%20configure%20string%20literal%20style%2C%20%23290%2C%20%23529.).
We can use `quotingType` and `forceQuotes`, but this will force the string style to be a specific way and we would need to change the expected files anyway.

Reply:
@aman-v-singh So with new version, what would these URLs be treated as? Are they not string anymore? I get that we have to change expected files but what would happen with users that have such files, would they be able to generate the collection? would there collection generated be any different?

Reply:
These are still `strings`. Only the bundled files in yaml format would be produced without any string quotes, and yes they would be able to generate collections. The generated collections will not be different.

Reply:
Cool, in that case it's fine.